charts/coturn/templates/configmap-coturn-conf-template.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "coturn.fullname" . }}
+  labels:
+    {{- include "coturn.selectorLabels" . | nindent 4 }}
+
+data:
+  coturn.conf.template: |
+    ## disable (d)tls control plane; don't permit relaying tcp connections.
+    no-tls
+    no-dtls
+    no-tcp-relay
+
+    ## don't turn on coturn's cli.
+    no-cli
+
+    ## authentication setup
+    # FUTUREWORK: enable support for multiple secrets?
+    zrest
+    static-auth-secret=__COTURN_SECRET__
```

Comment:
```suggestion
    static-auth-secret={{ .Values.secrets.zrestSecret | quote }}
```

Reply:
See my comments above regarding substituting values into the coturn config file.